docs/releases/stable/index.md

```diff
@@ -65,26 +65,54 @@ snaps as well as cdk-addons):
 - https://github.com/charmed-kubernetes/jenkins/pull/765/files
 - https://github.com/charmed-kubernetes/jenkins/commit/b0ce1fb0053908043ce25f10cca40be6531c3156
 
+Charm tracks can be created by contacting [~snapstore](https://chat.canonical.com/canonical/channels/snapstore) and asking for new tracks to be opened for every neccessary [charm](https://charmhub.io/charm) and [bundle](https://charmhub.io/bundle) owned by `Canonical Kubernetes` on [charmhub.io](https://charmhub.io)
+
 ## Preparing the release
 
 ### Tag existing stable branches with the current stable bundle
+#### :warning: **Deprecated Step**
+
+Starting with release 1.24, each repo has a unique branch for each 
+Charm release. This step was previously necessary to tag the 
+previous stable release before resyncing each branch from `main` -> `stable`.
+Tagging of the `release_x.xx` branch will now take place at the end of the release.
+
+> For all charm repos that make up CK, tag the existing stable branches with
+> the most recently released stable `cs:charmed-kubernetes` bundle revision.
+>
+> **Job**: https://jenkins.canonical.com/k8s/job/sync-stable-tag-bundle-rev/
+
+### Submit PR to bump K8S Track Map
+
+Add the next release to the track map enumerations. To use the newly created tracks, 
+include the next release to the track list/map.
 
-For all charm repos that make up CK, tag the existing stable branches with
-the most recently released stable `cs:charmed-kubernetes` bundle revision.
+```python
+    ...
+    ("1.25", ["1.25/beta", "1.25/edge"]),
+]
+```
 
-**Job**: https://jenkins.canonical.com/k8s/job/sync-stable-tag-bundle-rev/
+Example PR:
+ - https://github.com/charmed-kubernetes/jenkins/pull/974
 
-### Reset stable with master git branches
+This will allow and charm-builds targetting `edge` or `beta` channels to flow to the
+`1.25` tracks while any charm-builds targetting `candidate` or `stable` will flow to
+`1.24` tracks.
 
-Once all repositories are tagged, we need to reset stable branches with
-master. This will be our snapshot from which we test, fix, and subsequently
-promote to the new stable release.
+:warning: Nightly charm and bundle builds will target `latest/edge` and this `{track}/edge`
+
+### Reset release_x.xx from `default` git branches
+
+Once all repositories are tagged, we need to create release branches from
```

Comment:
Should we remove this once all repositories are tagged language here since the tagging does not happen until the end now?
```suggestion
We need to create release branches from
```